### PR TITLE
Make LaTeX repr a little prettier

### PR DESCRIPTION
**Pull request by ColCarroll**
Wraps the model representation in an `\begin{array}` environment, and variable names in `\text{...}`.

New: 
screen shot 2017-11-11 at 2 36 55 pm

Old:
screen shot 2017-11-11 at 2 39 18 pm

### CODE REVIEW

File: pymc3/model.py [hunk 1]
```diff
@@ -1140,7 +1147,7 @@ def _repr_latex_(self, name=None, dist=None):
             name = self.name
         if dist is None:
             dist = self.distribution
-        return self.distribution._repr_latex_(name=escape_latex(name), dist=dist)
+        return self.distribution._repr_latex_(name=r'\text{%s}' % name, dist=dist)
```
Comment (twiecki) [inline pymc3/model.py]: Don't we need to escape the name still?

**Comment by twiecki** [comment]
Looks much nicer! Would it look even better to center on the \sim?

**Comment by ColCarroll** [comment]
Aligned it with `{rcl}`.  I'm not attached to that if you'd prefer `{lcl}` or something.  

This newest one actually wraps all the variable names in \text properly so that it propagates.  Also replaced spaces in the arguments with `~` so that LaTeX respects them.  See the attached example with a tricky variable name displaying fine.

screen shot 2017-11-13 at 8 55 10 am

**Comment by junpenglao** [comment]
img

**Comment by twiecki** [comment]
Looks really nice! Merge once tests pass.

**Comment by fonnesbeck** [comment]
Yes, big improvement. Maybe one day we could get it to spit out a DAG.

**Pull request merged**